spec/routing/users_routing_spec.rb

```diff
@@ -1,31 +1,29 @@
-require "rails_helper"
+require 'rails_helper'
 
 RSpec.describe UsersController, type: :routing do
-  describe "routing" do
-
-    it "routes to #index" do
-      expect(:get => "/users").to route_to("users#index")
+  describe 'routing' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.